.github/workflows/build_package.yml

```diff
@@ -249,15 +297,24 @@ jobs:
       # code (no C/C++), so just build for one examplar python version on Linux.
       ##########################################################################
 
-      - name: Build TF Compiler Tools wheels
-        if: "matrix.build-package == 'py-tf-compiler-tools-pkg'"
+      - name: Build TF Compiler Tools wheels (Linux-Linux-x86_64)
+        if: "matrix.build-package == 'py-tf-compiler-tools-pkg' && matrix.build-family == 'linux-x86_64'"
         shell: bash
         run: |
           docker run --rm -w=/work \
             -v $PWD:/work \
             "${MANYLINUX_X86_64_IMAGE}" \
             bash -c 'export PATH=/opt/python/cp39-cp39/bin:$PATH; python ./c/build_tools/github_actions/build_dist.py py-tf-compiler-tools-pkg'
 
+      - name: Build TF Compiler Tools wheels (Linux-AArch64)
+        if: "matrix.build-package == 'py-tf-compiler-tools-pkg' && matrix.build-family == 'linux-aarch64'"
+        shell: bash
+        run: |
+          docker run --rm -w=/work \
+            -v $PWD:/work \
+            "${MANYLINUX_AARCH64_IMAGE}" \
+            bash -c 'export PATH=/opt/python/cp39-cp39/bin:$PATH; python ./c/build_tools/github_actions/build_dist.py py-tf-compiler-tools-pkg'
```

Comment:
Is an arm64 build necessary for the tf-compiler-tools? Those are pure Python, which might be portable across CPU architecture? You could try installing these from pip to see.

Reply:
Good question. I thought about it and decided to add it since I wasn't sure it was guaranteed to be pure Python going fwd. "Might as well...". I'll try though.